examples/llm_compression/openvino/tiny_llama/main.py

```diff
@@ -27,9 +26,9 @@ def main():
     dataset = load_dataset("wikitext", "wikitext-2-raw-v1", split="test")
 
     tokenizer = AutoTokenizer.from_pretrained(MODEL_ID)
-    model = OVModelForCausalLM.from_pretrained(MODEL_ID, export=True, load_in_8bit=False, compile=False, stateful=False)
+    model = OVModelForCausalLM.from_pretrained(MODEL_ID, export=True, load_in_8bit=False, compile=False)
```

Comment:
There are 3 more samples using it and conformance test

Reply:
GFI for examples https://github.com/openvinotoolkit/nncf/issues/3491. Taking into account that stateless and stateful models have to produce the same accuracy and I would suggest to postpone changes in the conformance test.